packages/client/src/rpc/helpers.ts

```diff
@@ -36,7 +36,7 @@ export function callWithStackTrace(handler: Function, debug: boolean) {
 /**
  * Returns tx formatted to the standard JSON-RPC fields
  */
-export const jsonRpcTx = (tx: TypedTransaction, block?: Block, txIndex?: number): JsonRpcTx => {
+export const jsonRPCTx = (tx: TypedTransaction, block?: Block, txIndex?: number): JSONRPCTx => {
```

Comment:
Can we maybe rename this one as well so that we do not need to keep this somewhat confusing lowercasing due to having this at the beginning, maybe just simply by renaming to something like `toJSONRPCTx()` or so?

Reply:
Yes, will do in a follow up